### PR TITLE
Use single `Object` type in whole project

### PR DESCRIPTION
**Pull request by cthulhu-rider**
Remove `Object` and `RawObject` types from `pkg/core/object` package.
Use `Object` type from NeoFS SDK Go library everywhere. Avoid using the
deprecated elements.

Signed-off-by: Leonard Lyubich <leonard@nspcc.ru>

### CODE REVIEW

File: pkg/core/object/object.go [hunk 1]
```diff
@@ -1,70 +1,18 @@
...
-		Object: obj,
-	}
-}
-
-// New constructs blank Object instance.
-func New() *Object {
-	return NewFromSDK(object.New())
-}
+// AddressFromObject returns address of the object.
+func AddressFromObject(obj *object.Object) *addressSDK.Address {
```
Comment (fyrchik) [inline pkg/core/object/object.go:9]: What about having an `Address()` in SDK?

Comment (fyrchik) [reply pkg/core/object/object.go:9]: Also, consider `AdressOf` as a name.

File: pkg/local_object_storage/writecache/writecache.go [hunk 2]
```diff
@@ -3,8 +3,8 @@ package writecache
 import (
 	"sync"
 
-	"github.com/nspcc-dev/neofs-node/pkg/core/object"
 	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/blobstor/fstree"
+	objectSDK "github.com/nspcc-dev/neofs-sdk-go/object"
```
Comment (fyrchik) [inline pkg/local_object_storage/writecache/writecache.go:7]: Can we just replace the import without an alias? There is no conflict.

**Review by fyrchik — APPROVED** [review]

File: pkg/core/object/object.go (hunk 1 above)
Comment (cthulhu-rider) [reply pkg/core/object/object.go:9]: > What about having an `Address()` in SDK?

I don't mind though it seems rather redundant due to its simplicity. In `node` it is very often used (as seen in current PR), but I have not yet seen a single use case in other projects.
More thoughts? @alexvanin @carpawell @KirillovDenis 

And I agree that `AddressOf` is more neat.

Comment (alexvanin) [reply pkg/core/object/object.go:9]: > I have not yet seen a single use case in other projects.

Client applications usually don't fetch address from object, indeed. Server applications (custom storage \ relay node implementations) can definitely benefit from that, but SDK is more client application oriented as for now.

File: pkg/local_object_storage/writecache/writecache.go (hunk 2 above)
Comment (alexvanin) [reply pkg/local_object_storage/writecache/writecache.go]: As far as I remember, @carpawell wanted to have the same import aliases for all SDK and V2 components? 
https://github.com/nspcc-dev/neofs-node/pull/1131#issuecomment-1031437700

Comment (carpawell) [reply pkg/local_object_storage/writecache/writecache.go]: ok, if there are no conflicts, it could be done in such way

**Review by alexvanin — APPROVED** [review]

**Pull request merged**